src/ui/qgslabelengineconfigdialog.ui

```diff
@@ -11,7 +11,7 @@
    </rect>
   </property>
   <property name="windowTitle">
-   <string>Automated Placement Engine</string>
+   <string>Project Label Settings</string>
```

Comment:
Isn't it the case that the dialog is also linked to diagram placement? If so, might be good to have a more generic title. The dialog _can_ be opened via the diagram's panel / dialog too.

Reply:
Ok, reverted